storage/gcp/dedup.go

```diff
@@ -57,28 +58,31 @@ type DedupStorage struct {
 var _ dedup.BEDedupStorage = &DedupStorage{}
 
 // Get looks up the stored index, if any, for the given identity.
-func (d *DedupStorage) Get(ctx context.Context, i []byte) (uint64, bool, error) {
-	var idx int64
-	if row, err := d.dbPool.Single().ReadRow(ctx, "IDSeq", spanner.Key{0, i}, []string{"idx"}); err != nil {
+func (d *DedupStorage) Get(ctx context.Context, i []byte) (dedup.SCTDedupInfo, bool, error) {
+	var idx, timestamp int64
+	if row, err := d.dbPool.Single().ReadRow(ctx, "IDSeq", spanner.Key{0, i}, []string{"idx", "timestamp"}); err != nil {
 		if c := spanner.ErrCode(err); c == codes.NotFound {
-			return 0, false, nil
+			return dedup.SCTDedupInfo{}, false, nil
 		}
-		return 0, false, err
+		return dedup.SCTDedupInfo{}, false, err
 	} else {
-		if err := row.Column(0, &idx); err != nil {
-			return 0, false, fmt.Errorf("failed to read dedup index: %v", err)
+		if err := row.Columns(&idx, &timestamp); err != nil {
+			return dedup.SCTDedupInfo{}, false, fmt.Errorf("failed to read dedup index: %v", err)
```

Comment:
```suggestion
			return dedup.SCTDedupInfo{}, false, fmt.Errorf("failed to read dedup index: %w", err)
```

Reply:
Is there any specific reason for this?
Switching to `%w` would make the underlying error, which is spanner specific, available to callers.
`Get` implements the standard `BEDedupStorage` interface from `modules/dedup`, and not all the `BEDedupStorage` implementations will use Spanner. So I think that no caller of `Get` should try to match the error type with a Spanner error type.